fbw-a380x/src/systems/instruments/src/MFD/shared/utils.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2023-2024 FlyByWire Simulations
+// SPDX-License-Identifier: GPL-3.0
+
+import { Approach, ApproachType } from '@flybywiresim/fbw-sdk';
+
+export function secondsToHHmmString(seconds: number) {
+  const minutesTotal = seconds / 60;
+  const hours = Math.abs(Math.floor(minutesTotal / 60))
+    .toFixed(0)
+    .toString()
+    .padStart(2, '0');
+  const minutes = Math.abs(minutesTotal % 60)
+    .toFixed(0)
+    .toString()
+    .padStart(2, '0');
+  return `${hours}:${minutes}`;
+}
+
+const approachTypeNames: Record<ApproachType, string> = {
+  [ApproachType.Ils]: 'ILS',
+  [ApproachType.Gls]: 'GLS',
+  [ApproachType.Igs]: 'IGS',
+  [ApproachType.Loc]: 'LOC',
+  [ApproachType.LocBackcourse]: 'BAC',
+  [ApproachType.Lda]: 'LDA',
+  [ApproachType.Sdf]: 'SDF',
+  [ApproachType.Gps]: 'GPS',
+  [ApproachType.Rnav]: 'RNV',
+  [ApproachType.Vor]: 'VOR',
+  [ApproachType.VorDme]: 'VOR',
+  [ApproachType.Vortac]: 'VOR',
+  [ApproachType.Ndb]: 'NDB',
+  [ApproachType.NdbDme]: 'NDB',
+  [ApproachType.Fms]: 'RNAV',
+  [ApproachType.Mls]: 'MLS', // not actually supported
+  [ApproachType.MlsTypeA]: 'MLS', // not actually supported
+  [ApproachType.MlsTypeBC]: 'MLS', // not actually supported
+  [ApproachType.Tacan]: 'TAC', // not actually supported
+  [ApproachType.Unknown]: '',
+};
+
+export function getApproachName(approach: Approach, withRnpSuffix = true): string {
```

Comment:
It would be cool to include this in `ApproachUtils.ts`, but I understand that unifying them is hard because of the many different formats.

Reply:
Yeah, in the end the two planes are actually quite different with this, so the other approach utils probably needs to move into fbw-a32nx/.